lib/mpl_toolkits/axes_grid1/inset_locator.py

```diff
@@ -420,14 +435,27 @@ def inset_axes(parent_axes, width, height, loc=1,
             'center'       : 10
 
     bbox_to_anchor : tuple or `matplotlib.transforms.BboxBase`, optional
-        Bbox that the inset axes will be anchored. Can be a tuple of
-        [left, bottom, width, height], or a tuple of [left, bottom].
+        Bbox that the inset axes will be anchored to. If None,
+        *parent_axes.bbox* is used. If a tuple, can be either
+        [left, bottom, width, height], or [left, bottom].
+        If *width* and/or *height* are given in relative units, the 2-tuple
+        [left, bottom] cannot be used. Note that
+        the units of the bounding box are determined through the transform
+        in use. When using *bbox_to_anchor* it almost always makes sense to
+        also specify a *bbox_transform*. This might often be the axes transform
+        *parent_axes.transAxes*.
 
     bbox_transform : `matplotlib.transforms.Transform`, optional
-        Transformation for the bbox. if None, `parent_axes.transAxes` is used.
+        Transformation for the bbox. If None, a `.transforms.IdentityTransform`
+        is used. This is useful when not providing any argument to
+        *bbox_to_anchor*. When using *bbox_to_anchor* it almost always makes
+        sense to also specify a *bbox_transform*. This might often be the
+        axes transform *parent_axes.transAxes*. Inversely, when specifying
+        the axes- or figure-transform here, make sure to provide a bounding
+        box to *bbox_to_anchor* as well, else the result may be unexpected.
```

Comment:
This is too vague.  What bbox is used if the user doesn't specify bbox_to_anchor?  Maybe: 

"...figure-transform here, be aware that not specifying *bbox_to_anchor* will use *parent_axes.bbox*, the units of which are in *figure.transFigure*"

Reply:
I'm now doing:

>Inversely, when specifying the axes- or figure-transform here, be aware that not specifying 
        *bbox_to_anchor* will use *parent_axes.bbox*, the units of which are
        in display (pixel) coordinates.